multimodal/src/autogluon/multimodal/utils/object_detection.py

```diff
@@ -14,7 +14,74 @@
 logger = logging.getLogger(AUTOMM)
 
 
-def from_dict(data):
+def convert_data_to_df(data) -> pd.DataFrame:
```

Comment:
I think we might need to add type descriptions here.

Reply:
good catch